### DISCUSSION TITLE
Square minimap option!!

### DISCUSSION THREAD

**Discussion by SerumFromThere** [runelite/runelite#17386] [Ideas]
Not sure if this is the place to do this exactly, but the idea of a squared minimap really has always been something I've really wanted in OSRS. As simple of a change as it would be, man it would look clean. --

Curious if the possibility is there and why Jagex hasn't done anything with the minimap over the years? The ability in other plugins already to zoom in and out far in the map itself makes it so a square version wouldn't give any sight advantages past what we already have.

**Comment by Psyda** [comment]
I've actually made this in the past while experimenting with what is possible with themes and it's very doable, but needs graphical elements created to fit the shape and while that would give you a square map it would introduce issues with the ORB elements overlapping the map, this would be better suited to a runelite plugin and would have compatability issues with the Resource Packs plugin

**Reply by TV-cmd** [reply]
> with the

Does this give an advantage to the rounded one? like is it bigger
